lib/aprb.ex

```diff
@@ -8,7 +8,7 @@ defmodule Aprb do
 
     children = [
       supervisor(Aprb.Repo, []),
-      worker(Aprb.Service.AmqEventService, [%{topic: "conversations"}], id: :conversations),
+      worker(Aprb.Service.AmqEventService, [%{topic: "conversations", routing_key: "conversation.*"}], id: :conversations),
```

Comment:
we are now going to get and event every time a message was added to a conversation with new push notification changes and with [this](https://github.com/artsy/impulse/pull/307) those messages will now go under `message.*` routing key, im filtering those here.